src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (...n) => {
```

Comment:
Nice thinking with arguments, but here task says that each call of `adder` function will have either one or no arguments.
So, you don't need to use rest operator and simply check `if (n === undefined)`